lblod/app-lokaal-mandatenbeheer
# Lmb 1053: consuming decision LDES

## Description

We want to consume the LDES that has all the decisions (besluit & artikel) on it. Use the dles-consumer for this. How the data will get to our database from here will be handled by the decision flow.

## How to test

1. Set a breakpoint in the mandataris service wher the decision flow starts
2. Run this app and it will consume the deployed LDES from app-mandatees-decisions
3. Breakpoint should stop the code

## Links to other PR's

- https://github.com/lblod/app-mandatees-decisions/pull/11
- https://github.com/lblod/mandataris-service/pull/66

## Other

![image](https://github.com/user-attachments/assets/7f47eb67-28a7-4cbd-bc55-5237588840b7)
